common/util.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
```

Comment:
Note: Similar functionality can be found in users/notifications.py when merging in the master branch.